### ISSUE TITLE
[BUG] ConsoleAliases are not properly usable

### ISSUE THREAD

**Issue by eqagunn** [ZDoom/Raze#1042]
### Raze version

Raze 1.8.1

### Which game are you running with Raze?

Duke Nukem 3D

### What Operating System are you using?

Windows 11

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Adding the following to raze.ini:
```
[Duke.ConsoleAliases]
Name=resurrect
Command=god; god
```
would and up getting split into:
```
[Duke.ConsoleAliases]
Name=Name
Command=resurrect
Name=Command
Command=god; god
```
upon starting the game and thus wouldn't work once in-game.

The current workaround is to modify your INI before **every** session with:
```
[Duke.ConsoleAliases]
resurrect=god; god
```
so it gets split into what it needs to be upon starting a game.

On a related note:

I was intending to bring GZDoom's `resurrect` command into Raze with this alias. Well, for Duke only since `god` doesn't revive in Blood. However, in Raze, while using `god` does revive, using `god` again to disable it leaves you with blocked movement and blocked horizontal look, with only vertical look working. Using `wait 100` in-between and having it bound to a key makes no difference.

Resurrection can be achieved via `god; god` in Duke WT and via `heal` in Blood FS. However, as per above, no way to do it in Duke Raze and Blood Raze that I'm aware of.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. 
2.
3.

**Comment by coelckers** [comment]
fixed in 4b6646da039267d80b08cb7cde279c44b3e46756

**Comment by eqagunn** [comment]
Are there any plans to bring GZDoom's `resurrect` command into Raze?
